GradienceTeam/Gradience
# Prevent set_gtk3_theme from crashing

## Description

When using `GSettingsSetting` outside a flatpak environment, the added getter, setter and reset methods should prevent the `AttributeError` when attempting to set the GTK3 theme.

Note: I used python-black for formatting the code and it made a lot of weird opinionated changes.

Fixes #811

## Type of change


## Changelog

- Adds a function to stage the `gsettings` command to be executed.
- Adds the equivalent setter, getter and reset functions for `GSettingsSetting`.

## Testing


### How to test the changes

Run `./local.sh` in a non-flatpak environment, apply a theme, making sure to check the GTK3 checkbox. The program should ask you to log out to see the effects.
